src/helpers.ts

```diff
@@ -58,20 +58,18 @@ export function codegenErrorReport(errors) {
         .join('\n');
 }
 
-export function formatErrors(instanceName: string, errors: ts.Diagnostic[]) {
-    return errors.map(function (diagnostic) {
-        let lineChar;
-        if (diagnostic.file) {
-            lineChar = diagnostic.file.getLineAndCharacterOfPosition(diagnostic.start);
-        }
-        return (
-            `[${ instanceName }] ` + (diagnostic.file ? diagnostic.file.fileName : '')
```

Comment:
Noted where it moved to. This just unwrapped `errors.map()`. Most of the diff is indentation.